R/rebuild.R

```diff
@@ -34,7 +34,9 @@ reset_search_path <- function() {
 }
 
 attach <- function(locked_package) {
-  if (!identical(locked_package$load, FALSE) && !locked_package$is_dependency_package) {
+  if (is.autoinstall_package(locked_package)) {
```

Comment:
How does `load` interact with `autoinstall`? Presumably, if `load = FALSE`, we should simply re-install, but not re-load, correct?